### PR TITLE
CB-1926: Update blueprint for SDX-HA lite using cardinality

### PR DESCRIPTION
**Pull request by kkalvagadda**
Used cardinality to create lite SDX HA cluster following the lines of blueprint for medium SDX HA.

Verified the changed by creating an SDX cluster in Manowar-int using it.

### CODE REVIEW

File: core/src/main/resources/defaults/blueprints/cdp-sdx-ha.bp [hunk 1]
```diff
@@ -374,31 +308,43 @@
...
       },
       {
-        "serviceType": "KNOX",
         "refName": "knox",
         "roleConfigGroups": [
           {
             "base": true,
+            "configs": [
+              {
+                "name": "gateway_master_secret",
```
Comment (adoroszlai) [inline core/src/main/resources/defaults/blueprints/cdp-sdx-ha.bp]: Please don't add these hard-coded values in `master`.  There is no need for them in recent CB versions.  If you would like to use the blueprint in environment running older CB, just add the secret in the file you use to create the blueprint there.

Comment (kkalvagadda) [reply core/src/main/resources/defaults/blueprints/cdp-sdx-ha.bp]: Does CB in mow-int inject this configuration automatically now?

Comment (adoroszlai) [reply core/src/main/resources/defaults/blueprints/cdp-sdx-ha.bp]: Yes.

**Comment by lacikaaa** [comment]
Also note that our jenkins CI jobs can run only if you push your branch to the hortonworks repo, we can't run it on forks. So please migrate your commit to a branch from this repo and open the PR from it

**Comment by adoroszlai** [comment]
Obsoleted by #5562.

**Pull request closed without merging**